lib/utils.pm

```diff
@@ -1196,13 +1196,9 @@ sub reconnect_s390 {
         select_console('iucvconn');
     }
     else {
-        my $r = wait_serial($login_ready, 300);
-        if ($r && $r =~ qr/Welcome to SUSE Linux Enterprise 15/) {
-            record_soft_failure('bsc#1040606');
```

Comment:
The bug is RESOLVED FIXED. Could you please make it VERIFIED FIXED before removing this soft fail reference?

Reply:
As we've already discussed with you some time ago, the ticket number, mentioned on 1201 line is wrong and not related to the workaround with login message on s390x. 

Also, I don't know why that workaround itself was added as it was never executed after adding. Looks like an attempt to make a workaround for bsc#1083506.